sdk/keyvault/azure-security-keyvault-keys/src/key_client.cpp

```diff
@@ -26,8 +26,13 @@ KeyClient::KeyClient(
   policies.emplace_back(std::make_unique<TelemetryPolicy>("KeyVault", apiVersion));
   policies.emplace_back(std::make_unique<RequestIdPolicy>());
   policies.emplace_back(std::make_unique<RetryPolicy>(options.RetryOptions));
-  policies.emplace_back(std::make_unique<BearerTokenAuthenticationPolicy>(
-      credential, "https://vault.azure.net/.default"));
+
+  {
+    Azure::Core::Http::GetTokenOptions const tokenOptions = {{"https://vault.azure.net/.default"}};
+    policies.emplace_back(
+        std::make_unique<BearerTokenAuthenticationPolicy>(credential, tokenOptions));
```

Comment:
Would this work: ???

```suggestion
    policies.emplace_back(
        std::make_unique<BearerTokenAuthenticationPolicy>(credential, {{"https://vault.azure.net/.default"}}));
```

Reply:
Did not compile on MSVC for me :(